addons/mrp/wizard/mrp_product_produce.py

```diff
@@ -43,6 +43,13 @@ def default_get(self, fields):
     workorder_line_ids = fields.One2many('mrp.product.produce.line', 'product_produce_id')
     move_raw_ids = fields.One2many(related='production_id.move_raw_ids')
 
+    def _update_workorder_lines(self):
+        res = super(MrpProductProduce, self)._update_workorder_lines()
```

Comment:
Why does the quantity_done is zero. It seems that it's not intended
https://github.com/odoo/odoo/blob/master/addons/mrp/models/mrp_abstract_workorder.py#L139
https://github.com/odoo/odoo/blob/master/addons/mrp/models/mrp_abstract_workorder.py#L152
https://github.com/odoo/odoo/blob/master/addons/mrp/models/mrp_abstract_workorder.py#L162